InternetShop/InternetShop/Startup.cs

```diff
@@ -27,6 +30,17 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
 
+            services.AddBusinessLogic(Configuration);
+
+            var mapperConfig = new MapperConfiguration(mc =>
```

Comment:
Replace to typeof(Mappers.MappingProfile).Assembly, typeof(BLL.Mappers.MappingProfile).Assembly);

Reply:
services.AddAutoMapper(x =>
                {
                    x.AddExpressionMapping();
                },typeof(Mappers.MappingProfile).Assembly, typeof(BLL.Mappers.MappingProfile).Assembly);